src/components/layout/parts/head.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Helmet from 'react-helmet'
+import { useStaticQuery, graphql } from 'gatsby'
+
+function HEAD({
+  description,
+  lang,
+  meta,
+  keywords,
+  title,
+  locales,
+  landingName
+}) {
+  const { site } = useStaticQuery(
+    graphql`
+      query {
+        site {
+          siteMetadata {
+            title
+            description
+            author
+            siteURL
+            defaultLocale
+          }
+        }
+      }
+    `
+  )
+  const metaTitle = title || site.siteMetadata.title
+  const metaDescription = description || site.siteMetadata.description
+  const metaLocales = locales || site.siteMetadata.defaultLocale
+  const metaLandingName = landingName || ''
+
+  return (
+    <Helmet
+      htmlAttributes={{
+        lang
+      }}
+      title={metaTitle}
+      titleTemplate={`%s | ${site.siteMetadata.title}`}
+      meta={[
```

Comment:
Could you please create a file called "meta" in the same directory and move this to it